DragaliaAPI/AutoMapper/Profiles/FortMapProfile.cs

```diff
@@ -10,6 +10,7 @@ public FortMapProfile()
     {
         this.CreateMap<DbFortBuild, FortPlantList>();
         this.CreateMap<DbFortBuild, BuildList>();
+        this.CreateMap<DbFortDetail, FortDetail>();
```

Comment:
Instead of providing an operator in the generated class, you could do

```suggestion
        this.CreateMap<DbFortDetail, FortDetail>()
        .ForMember(x => x.working_carpenter_num, opts => opts.MapFrom(src => 5));
```

Reply:
Actually I've just seen the test pipeline -- you will have to do this to stop automapper complaining :innocent: